cli/install/install.go

```diff
@@ -361,6 +389,14 @@ func checkExisting(version string, dst string) bool {
 	}
 }
 
+// isVersion сhecks the string for
+// possible compliance with the version,
+// since according to the conditions,
+// the version format looks like X.Y.Z.
+func isVersion(version string) bool {
+	return strings.Contains(version, ".")
```

Comment:
Why not use `version.Parse()` from version_tools.go to check this?

Reply:
fixed